generated/java/gapic-google-cloud-pubsub-v1/src/main/java/com/google/cloud/pubsub/v1/TopicAdminClient.java

```diff
@@ -274,6 +277,61 @@ private final Topic createTopic(Topic request) {
 
   // AUTO-GENERATED DOCUMENTATION AND METHOD
   /**
+   * Updates an existing topic. Note that certain properties of a topic are not modifiable. Options
+   * settings follow the style guide: NOTE: The style guide requires body: "topic" instead of body:
+   * "&#42;". Keeping the latter for internal consistency in V1, however it should be corrected in
+   * V2. See https://cloud.google.com/apis/design/standard_methods#update for details.
+   *
+   * <p>Sample code:
+   *
+   * <pre><code>
+   * try (TopicAdminClient topicAdminClient = TopicAdminClient.create()) {
+   *   Topic topic = Topic.newBuilder().build();
+   *   FieldMask updateMask = FieldMask.newBuilder().build();
+   *   UpdateTopicRequest request = UpdateTopicRequest.newBuilder()
+   *     .setTopic(topic)
+   *     .setUpdateMask(updateMask)
+   *     .build();
+   *   Topic response = topicAdminClient.updateTopic(request);
+   * }
+   * </code></pre>
+   *
+   * @param request The request object containing all of the parameters for the API call.
+   * @throws com.google.api.gax.grpc.ApiException if the remote call fails
+   */
+  /* package-private */ final Topic updateTopic(UpdateTopicRequest request) {
```

Comment:
What exactly? The update for Pubsub? Then yes, otherwise I would not be able to generate batch. The update (which includes proto files changes with new Update* methods in them for pubsub) was recently pushed (automatically by sync job) to googleapis.